.github/workflows/test-ubsan.yml

```diff
@@ -45,19 +45,19 @@ jobs:
       LINK: g++
       CONFIG_FLAGS: --enable-ubsan
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11  # v4.1.1
         with:
           persist-credentials: false
       - name: Store suppressions path
         run: |
           echo "UBSAN_OPTIONS=suppressions=$GITHUB_WORKSPACE/suppressions.supp" >> $GITHUB_ENV
       - name: Set up Python ${{ env.PYTHON_VERSION }}
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@0a5c61591373683505ea898e09a3ea4f39ef2b9c  # v5.0.0
         with:
           python-version: ${{ env.PYTHON_VERSION }}
       - name: Environment Information
         run: npx envinfo
       - name: Build
         run: make build-ci -j2 V=1
```

Comment:
This line can be `-j4` as well

Reply:
Look at the previous commits. It fails to compile with -j4 and -j3.